account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -186,15 +192,19 @@ def update_product_taxes(self, product, taxes_field, company_from):
         return False
 
     def match_tax(self, tax_template):
-        if not tax_template.description:
+        """We can only match the new company tax if the chart was used"""
```

Comment:
When is this method used? It seems non covered by tests.

Reply:
It's used for the default company taxes (purchase and sales)

Reply:
And can't be used the same or cover it by tests?